statusHandler/presenter/instanceInfoGetters.go

```diff
@@ -41,3 +52,83 @@ func (psh *PresenterStatusHandler) GetCountLeader() uint64 {
 func (psh *PresenterStatusHandler) GetCountAcceptedBlocks() uint64 {
 	return psh.getFromCacheAsUint64(core.MetricCountAcceptedBlocks)
 }
+
+// GetNodeName will return node name
+func (psh *PresenterStatusHandler) GetNodeName() string {
+	nodeName := psh.getFromCacheAsString(core.MetricNodeDisplayName)
+	if nodeName == "" {
+		nodeName = "noname"
+	}
+
+	return nodeName
+}
+
+// GetTotalRewardsValue will return total value of rewards and how rewards was increased on every second
+// Rewards estimation will be equal with :
+// (numSignedBlocks - numProposedBlocks) * communityPercentage * Rewards +
+// numProposedBlocks * leaderPercentage * Rewards
+func (psh *PresenterStatusHandler) GetTotalRewardsValue() (string, string) {
+	rewardsValue := psh.getBigIntFromStringMetric(core.MetricRewardsValue)
+	communityPercentage := psh.getFloatFromStringMetric(core.MetricCommunityPercentage)
+	leaderPercentage := psh.getFloatFromStringMetric(core.MetricLeaderPercentage)
+	numSignedBlocks := psh.getFromCacheAsUint64(core.MetricCountConsensusAcceptedBlocks)
+	numProposedBlocks := psh.getFromCacheAsUint64(core.MetricCountAcceptedBlocks)
+
+	rewardsCommunityCoefficient := math.Abs(float64(int(numSignedBlocks-numProposedBlocks))) * communityPercentage
+	consensusSignedBlocksRewards := big.NewFloat(rewardsCommunityCoefficient)
+	consensusSignedBlocksRewards.Mul(consensusSignedBlocksRewards, big.NewFloat(0).SetInt(rewardsValue))
+
+	rewardsLeaderCoefficient := float64(numProposedBlocks) * leaderPercentage
+	leaderProposedBlocksRewards := big.NewFloat(rewardsLeaderCoefficient)
+	leaderProposedBlocksRewards.Mul(leaderProposedBlocksRewards, big.NewFloat(0).SetInt(rewardsValue))
+
+	totalRewardsBigFloat := big.NewFloat(0).Add(consensusSignedBlocksRewards, leaderProposedBlocksRewards)
+	totalRewards := new(big.Int)
+	totalRewardsBigFloat.Int(totalRewards)
+
+	difRewards := big.NewInt(0).Sub(totalRewards, psh.totalRewardsOld)
+	psh.totalRewardsOld.Set(totalRewards)
+	totalRewards.Sub(totalRewards, difRewards)
+
+	return totalRewards.Text(10), difRewards.Text(10)
+}
+
+// CalculateRewardsPerHour will return an approximation of how many test ERDs a validator will earn per hour
```

Comment:
This code should not know about being in the testnet. So we can remove the "test" from the comments.

Reply:
Done.